RELEASE.md

```diff
@@ -172,24 +172,21 @@ Depending on the type of release, run one of the following:
 * `make release-major`: This will release the next major version. Use this to if the changes modify the user API in a backwards incompatible way after the major version 1 has been released.
```

Comment:
this is a little misleading. If you run `make release` and the version has already been bumped to a minor or major version, it will release that version.

Reply:
Fixed:
https://github.com/sdv-dev/Copulas/pull/485/commits/af635acf4b9147ae1852db5893d7de6a1d723a64